### PR TITLE
tests: more fixes for e2e

### PR DESCRIPTION
**Pull request by gssbzn**
## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: e2e/atlas/access_logs_test.go [hunk 1]
```diff
@@ -35,18 +35,17 @@ func TestAccessLogs(t *testing.T) {
...
-		t.Fatalf("unexpected error: %v", err)
-	}
-
 	defer func() {
 		if e := deleteCluster(clusterName); e != nil {
 			t.Errorf("error deleting test cluster: %v", e)
 		}
 	}()
 
+	h, err := getHostname()
```
Comment (gssbzn) [inline e2e/atlas/access_logs_test.go:44]: if this fails, make sure to have defined the clean up function for project

File: e2e/atlas/helper_test.go [hunk 2]
```diff
@@ -131,7 +132,7 @@ func getProcesses() ([]*mongodbatlas.Process, error) {
 	return processes, nil
 }
 
-func deployCluster() (string, error) {
+func deployClusterForProject(projectID string) (string, error) {
```
Comment (gssbzn) [inline e2e/atlas/helper_test.go:135]: reduce flakyness by deploying per project

**Review by fmenezes — APPROVED** [review]
the code looks good, check evergreen

**Comment by gssbzn** [comment]
>check evergreen

Just merged master, waiting for it to pass with all cluster test patched

**Pull request merged**